rocket_sqlite_oracle/src/db/kraken.rs

```diff
@@ -257,7 +257,7 @@ mod wire {
 
     #[derive(Debug, Clone)]
     pub struct USDPrice {
-        pub millicents: u64,
+        pub exchange_rate: u64,
```

Comment:
🙃 You might want to use a `tuple struct` in situations where you have a `struct` with a single field. As in:

```rust
struct UsdPrice(pub u64);
```